extensions/HyperVExtension/src/HyperVExtension/Constants.cs

```diff
@@ -28,4 +28,8 @@ internal sealed class Constants
     public const string HyperVTemplatesSubPath = @"HyperVExtension\Templates";
 
     public static string SystemRootPath { get; } = Path.GetPathRoot(Environment.SystemDirectory)!;
+
+    public static int GuestPlatformIdWindows { get; } = (int)Windows.Win32.System.Diagnostics.Debug.VER_PLATFORM.VER_PLATFORM_WIN32_NT;

```

Comment:
For `Windows.Win32.System.Diagnostics`, should we put this in a using statement?

Reply:
Was one time thing. I'll add it for "Windows.Win32.System.Diagnostics.Debug"